app/api-v1/api-doc.js

```diff
@@ -184,6 +213,7 @@ const apiDoc = {
             description: 'Name of the supplier who ran the build. This information is not stored directly on-chain',
             type: 'string',
             maxLength: 255,
+            example: 'SupplierAlias',
           },
           status: {
```

Comment:
Do the `date-time` fields such as `completionEstimatedAt` not need examples?

Reply:
the swagger interface auto populates them with the current time. I could add examples for the ones where current time doesn't make sense e.g. `completionEstimatedAt` but not bother for `submittedAt` etc.